metrics/Code_Changes_Lines.md

```diff
@@ -88,9 +69,9 @@ and likely, actor merging.
 These could be represented as bar charts, with time running in the X axis.
 Each bar would represent a code changes during a certain period (eg, a month).
 
-## Reference Implementation
- 
-## Known Implementations
+
+### Tools Providing the Metric
+
 * [GrimoireLab](https://chaoss.github.io/grimoirelab) provides this metric out of the box.
   - View an example on the [CHAOSS instance of Bitergia Analytics](https://chaoss.biterg.io/app/kibana#/dashboard/f13af0e0-18e5-11e9-ba47-d5cbef43f8d3).  
   - Download and import a ready-to-go dashboard containing examples for this metric visualization from the [GrimoireLab Sigils panel collection](https://chaoss.github.io/grimoirelab-sigils/chaoss-gmd-cde/lines_of_code_changed/).
```

Comment:
Are these visualizations that could be added to the visualization sub-heading above?

Reply:
For now the effort is only moving stuff around, not adding things. We can definitely add things as well.